docs: add some info regarding pg roles (#335)
graphile/worker: README.md

(`npx` looks for the `graphile-worker` binary locally; it's often better to use
the `"scripts"` entry in `package.json` instead.)

> Note: Graphile Worker expects the runtime postgres role to be the same as the one used while running the migrations. If for some reason you need to run your migrations as a different role, you can fix this by changing the owner role of the tables to be the runtime one.

Reviewer: ```suggestion
Note: Graphile Worker expects the Postgres role used at runtime to be the same
as the role used while running the migrations. If you need to run your
migrations as a different role, one solution is to explicitly change the owner
of the `graphile_worker.*` tables to be the same role as is used at runtime.
```

Reviewer: I used Prettier Playground in markdown/prose wrap always mode to format this text:

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA5C8kAIDiAnAQwAcALASwBs4sB1CPAazjyzgA8i4wYBnLGEtQAKEHjADmeOHzwQqWAK484AEywEYWPAthkAttRgQsAI0OCsPAgfV8B1WfKWqsAd3LztUKGSjj+Fnpkkhpk0DwAdFgAkgBmWACeEApYUHAuRlo6icksQSEwYVB8BCVYKmSxscwImo5wADRY0NQ8cgqF0FhkdsbsRBRkYGQwFAlYYCQEfubUEK5pLBDx9lgABiGklHAA+q70THgRAFRr-AQmVL2ms5bWDnLUpd18zmoaWboGESANIBBETrFZCgAh4WSuIRghA8ZAgAgUVwEBKwv4mQhgJgwADK9wAMr44MhYgjlGiMVjsUQCMM-MgYNpGiA4HozCoVKo8dNxAoCOI4AAxeh6DSFOkoAgdCC-EAkGB6Cg0cjwHjUsBwbEwkZkABuIwScLAPFRIF8yjwMCEhHEIuJpKZACseGxsb5xFQAIoKDBEpAkihkkDUvDmuEixgqeZQGVEPC+GA0MgqATIAAcAAY-rHRHAaIQiHCEUiUTKAI7e+BWgGwiU8AC0aVUqhlUnLZCkVr5tr99r+yiC9MZfbdVAAgjAGWQTB04EJmAS0naA0yeCO4F6fUvAzALonkyRkAAmP4MgiUN0AYQgem7zJ4AFYZc4ACoXGv+wM6hRwaJQDmwbEwDjQFRz-bEYASKgtzgABfGCgA
